src/routes/landing/components/LocationSelector.css

```diff
@@ -0,0 +1,21 @@
+@import url('../../../modules/common/constants/colors.css');
+
+.language-list-item {
+  display: block;
+  padding: 7px;
+  color: inherit;
+  text-decoration: inherit;
+
+  &:hover {
+    color: inherit;
+    text-decoration: inherit;
+    transition: background-color 0.5s ease;
+    background-color: var(--hover-color);
+  }
+}
+
+.language-list-parent {
+  height: var(--item-height);
+  line-height: var(--item-height);
```

Comment:
--item-height is not yet defined here (see above).

Reply:
There are no warning when something is not defined in css?

Reply:
We should take a look at: https://github.com/styled-components/styled-components
Makes more sense than having real .css files which are then compiled to js modules. We could write all our css directly in js, which also provies a lot more possibilities.